components/edit-collective/EditHostInvoice.js

```diff
@@ -43,33 +51,46 @@ const EditHostInvoice = ({ collective }) => {
           {getErrorFromGraphqlException(setValue).message}
         </MessageBox>
       )}
-      <Flex flexWrap="wrap">
+      <Flex flexWrap="wrap" flexDirection="column">
         <StyledInput
           placeholder="Payment Receipt"
           defaultValue={value}
           onChange={e => setValue(e.target.value)}
           width="100%"
           maxWidth={300}
+          my={3}
+        />
+
+        <StyledTextarea
+          placeholder="Add any other text to appear on payment receipts, such as your organization's tax ID number, info about tax deductibility of contributions, or a custom thank you message."
+          defaultValue={info}
+          onChange={e => setInfo(e.target.value)}
+          width="100%"
+          height="150px"
+          maxWidth={300}
           my={2}
         />
+
+        {/* <textarea placeholder="Add any other text to appear on payment receipts, such as your organization's tax ID number, info about tax deductibility of contributions, or a custom thank you message."defaultInfo={info} onChange={e => setInfo(e.target.value)}> </textarea> */}
+
         <StyledButton
           buttonStyle="primary"
-          mx={2}
           my={2}
+          maxWidth={300}
           minWidth={200}
           loading={loading}
           maxLength={255}
-          disabled={!isTouched}
+          disabled={!isTouched && !isTouch}
           onClick={() =>
             setSettings({
               variables: {
                 id: collective.id,
-                settings: { ...collective.settings, invoiceTitle: value },
+                settings: { ...collective.settings, invoiceTitle: value, extraInfo: info },
```

Comment:
As per the [specs](https://github.com/opencollective/opencollective/issues/2907#issuecomment-589586209) the field should be `settings.invoice.extraInfo` (not `settings.extraInfo`).

Reply:
This has been marked as resolved but the code hasn't changed

Reply:
i thought i pushed the changes before clicking resolve, my mistake. Does this my change fix work?